### ISSUE TITLE
Beneficiary travel health care page leftover legacy rewriting work

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#4804]
## User Story:
As a Veteran, I want to understand how to get reimbursed for travel to a VA medical appointment.

Background for this ticket can be found in old repo in this original ticket: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/16955

This page was originally built in Github and sent to SMEs in spring 2019. It needs to be rebuilt in Drupal and shared with SMEs again since so much time has passed. Also, a few changes need to be made to the page per design rules and Drupal anomalies: 
- Remove the "On this page" feature 
- Make featured content box shorter

## Tasks

- [x] Build page from Heroku instance in Drupal - 4 hour(s) - @RLHecht 
- [ x] Rework page to eliminate Drupal anomalies - 2 hour(s) - @AnneHurley 
- [x] Share new page with stakeholders and implement SME feedback - 5 hour(s) - @AnneHurley 
- [ ] Final copyedit and publish page in Drupal - 1 hour - @RLHecht 

## Acceptance Criteria:

* Validated on live site

## Configuration
- [x] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (e.g. `vsa-authenticated-exp`, `vsa-caregiver`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by RLHecht** [comment]
@AnneHurley I built this page in Drupal from the Heroku, but there are some things that need to be changed before we can send it to the SMEs. We have to remove the "On this page" text that was below the intro, which I've already done. We also need to make the featured content section shorter. Happy to discuss when you're back before you get started on it if you have questions. Here's the preview link: preview-prod.vfs.va.gov/preview?nodeId=2431

**Comment by RLHecht** [comment]
@AnneHurley I took the featured content out of the blue box so it's now set up as a regular content block. This is now ready for you to tweak the H2s and anything else you need to. Preview link: preview-prod.vfs.va.gov/preview?nodeId=2431

**Comment by AnneHurley** [comment]
@RLHecht  H2s are updated, see what you think.

**Comment by AnneHurley** [comment]
@jenniferlee-dsva  Can you take a quick peek at this page: preview-prod.vfs.va.gov/preview?nodeId=2431

Want to make sure the H2s look all right to you. We changed some headlines and sizes after we eliminated the blue highlight box, which was too big. 
After you OK this I will send to the stakeholder/s (and can you remind me which ones they are for this?)

thanks!

**Comment by jenniferlee-dsva** [comment]
@AnneHurley header hierarchy looks good. 2 minor comments: 

1/ the first question (How do I find out if I'm eligible) doesn't quite seem right/to match the answer. Maybe it's more 'what are the eligibility requirements for VA travel pay reimbursement?' or 'am I eligible for VA travel pay reimbursement?' so something similar. 

2/The header before the subway steps (For each scheduled appointment) - For some reason, it seemed not associated with the steps and at first I definitely wondered, 'does this mean I have to submit a form for every single appointment?'  And even though it's also in the body copy of that step 1 text, I scanned it and missed it first. 

Can we make this very very super duper clear by just incorporating the current subheader info right into the step 1 header.  It might seem a little redundant with the body text, but I feel like many people (like I did) will just scan the header text and links without closely reading the body text, and this seems like an important (and not totally expected) requirement.

Like this:

**(1) Fill out a reimbursement claim for each appointment** 
You’ll need to fill out a new Veteran/Beneficiary Claim for Reimbursement of Travel Expenses (VA Form 10-3542) for each appointment. 
Download VA Form 10-3542 (PDF)

**Comment by AnneHurley** [comment]
@RLHecht  @jenniferlee-dsva  Fixes made to: http://prod.cms.va.gov/va-travel-pay-reinbursement. 

Sending to SMEs again now.

**Comment by AnneHurley** [comment]
@RLHecht  This is ready for your review and publish, and @lunascoop  it's ready to be redirected. (I can start the redirect process if that hasn't happened already, let me know)

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/3040

Here is the SME feedback:
1.  Add 2 additional eligibilities for Veteran eligibility:
 
•	A Veteran traveling to obtain a service dog under 38 CFR 17.148.
 
•	Veterans with vision impairment, spinal cord injury or disorder (SCI/D), or with double or multiple amputations whose travel is in connection with care provided through a special disabilities rehabilitation program of the Department (including programs provided by spinal cord injury centers, blind rehabilitation centers, and prosthetics rehabilitation centers) if such care is provided:
 
1.	on an in-patient basis; or
 
2.	during a period in which the Secretary provides the Veteran with temporary lodging at a facility of the Department to make such care more accessible to the Veteran.
 
2. MUST remove the following paragraph (this is NOT proper procedure for external or internal audience at this time):
 
BEGIN DELETE
How do I get reimbursed for travel by ambulance?
•	We have a special system to handle your ambulance claims. It’s called the fee-based claims system, and it helps us process your claims quickly so we can send your reimbursement faster.
For more information, please email us at VHABeneficiaryTravelQuestions@va.gov.
END DELETE

What I did:
Added a bullet about getting a service dog to the top bullet list
Added a section in that top block, under the "NOTE" about qualifying if you're traveling to a special kind of treatment center, like spinal cord rehab, vision loss rehab, etc. There are two bullets under that scenario, so I felt it should be a separate section. But let me know if you think how I have it works - maybe it would look better to have it right under the bullets and above the "NOTE."?

Also deleted the whole section about being reimbursed for ambulance travel, which appears under the subway map. The spacing above and below the horizontal line looks inconsistent but I couldn't see how to change it - if you can/do, can you tell  me how?

Lastly, Howie couldn't figure out that "editorial notes" field being missing, so I did a workaround in the little "log notes" field under me being the last user, but they are pretty abbreviated. 

Let me know if you have any questions, and if not this is ready to publish.

**Comment by RLHecht** [comment]
@AnneHurley I did a final read on this page and published it though I have one question about a link, which we can change. The link for the American Public Transportation Association wasn't working so I changed it to this: https://www.apta.com/research-technical-resources/public-transportation-links/ If you think it should just go to the home page, let me know. Though I think this provides the information we are describing in the text. 

Also I changed the name of the National Center on Senior Transportation to the National Aging and Disability Center to match what their website uses.

**Comment by AnneHurley** [comment]
@RLHecht 
Thank you for checking that link and that name. I think you made the right call with both. When the page is live I'll send out the final to Jeff and team, thanks!

**Comment by RLHecht** [comment]
This is validated on live site. Closing issue.

**Comment by jenniferlee-dsva** [comment]
Woo hoo! @RLHecht and @AnneHurley - just a quick fyi. On the health care hub page, the travel link still points to the legacy .asp URL.

**Comment by RLHecht** [comment]
@jenniferlee-dsva Yeah, I noticed on Friday afternoon that the menu items hadn't been set up and I disabled the legacy on on the left nav so it should all be good by today's deploy. I'll double check this afternoon.
